amazon-qldb-driver-core/src/error.rs

```diff
@@ -1,10 +1,8 @@
-use aws_sdk_qldbsession::{error, SdkError};
-
-use core::fmt;
 use aws_smithy_http::operation::BuildError;
 use thiserror::Error;
 
 pub type QldbResult<T> = std::result::Result<T, QldbError>;
+pub type BoxError = Box<dyn std::error::Error + Send + Sync + 'static>;
```

Comment:
Why include the word `Box` in the type name? Isn't the type signature enough for the reader to know this is a Box, if that is important for them to know?

Reply:
This has become idiomatic in libraries. I'm not sure what a better name might be, so I'm copying a pattern I've seen elsewhere. One example is the aws sdk itself.